wasm/src/debug_service.rs

```diff
@@ -267,137 +264,130 @@ impl From<StepResultId> for usize {
     }
 }
 
-#[wasm_bindgen]
-#[derive(Serialize, Deserialize)]
-struct StructStepResult {
-    pub id: usize,
-    pub value: usize,
-}
-
-#[wasm_bindgen(typescript_custom_section)]
-const IStructStepResult: &'static str = r#"
-export interface IStructStepResult {
-    id: number;
-    value: number;
-}
-"#;
-
-#[wasm_bindgen(typescript_custom_section)]
-const IBreakpointSpanList: &'static str = r#"
-export interface IBreakpointSpan {
-    id: number;
-    lo: number;
-    hi: number;
-}
-
-export interface IBreakpointSpanList {
-    spans: Array<IBreakpointSpan>
-}
-"#;
-
-#[derive(Serialize, Deserialize)]
-struct BreakpointSpanList {
-    pub spans: Vec<BreakpointSpan>,
+serializable_type! {
+    pub struct StructStepResult {
```

Comment:
`StructStepResult` seems to be the only `pub struct`, while the others are just `struct`. Is this on purpose?

Reply:
Good call out. I was in the middle of trying to figure out whether we actually needed `pub`, then I forgot. I think they can all be made private.

Reply:
I see now that a couple of the types under wasm/src/language_service.rs also have the `pub`.